packages/webview_flutter/tizen/src/buffer_pool.cc

```diff
@@ -102,3 +117,22 @@ void BufferPool::Prepare(int width, int height) {
     buffer->Reset(width, height);
   }
 }
+
+#include <cairo.h>
+void BufferUnit::dumpToPng(int filename) {
+  char filePath[256];
+  sprintf(filePath, "/tmp/dump%d.png", filename);
+  tbm_surface_info_s surfaceInfo;
+  tbm_surface_map(tbm_surface_, TBM_SURF_OPTION_WRITE, &surfaceInfo);
+  void* buffer = surfaceInfo.planes[0].ptr;
+
+  cairo_surface_t* png_buffer;
+  png_buffer = cairo_image_surface_create_for_data(
+      (unsigned char*)buffer, CAIRO_FORMAT_ARGB32, width_, height_,
+      surfaceInfo.planes[0].stride);
+
+  cairo_surface_write_to_png(png_buffer, filePath);
+
+  tbm_surface_unmap(tbm_surface_);
+  cairo_surface_destroy(png_buffer);
+}
```

Comment:
Are you sure you want to add this? It looks like a temporary implementation for debugging during development.

Reply:
It's intentional thing. Yes it's for debugging purposes first but I added it because I think it's useful enough for various purposes.

Reply:
How about using #ifndef NDEBUG

Reply:
Ok. I can do that.